### PR TITLE
Add ALPINE omnidb version

### PR DESCRIPTION
**Pull request by thomasboussekey**
Reduced image size from 819MB to 422MB

### CODE REVIEW

**Comment by thomasboussekey** [comment]
Add port exposure for websocker: 25482

File: alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      && mkdir /etc/omnidb 
+#      && apk del .build-deps \
+#      && rm -rf /var/lib/apt/lists/*
+RUN cd /opt/OmniDB-${OMNIDB_VERSION} && pip3 install --upgrade pip==9.0.3 && echo "Begin install cherrypy" && pip3 install cherrypy && echo "Begin install requirements" && pip3 install -r requirements.txt
+#RUN cd /opt/OmniDB-${OMNIDB_VERSION} && pip3 install pip --upgrade
+#RUN cd /opt/OmniDB-${OMNIDB_VERSION} && pip3 install cherrypy 
+#RUN cd /opt/OmniDB-${OMNIDB_VERSION} && pip3 install -r requirements.txt
+
+# Cleanup
+# RUN apt-get remove -y curl unzip
```
Comment (taivokasper) [inline alpine/Dockerfile]: Could you remove the lines that are commented out?

**Comment by thomasboussekey** [comment]
Hello @taivokasper ,

Should `alpine` replace `debian`??? I'm not sure. `Alpine` is a light-weight linux OS optimized for contenerization. But offering both OS can be very interesting.

Is-it possible to modify CI/CD to tags the 2 OS and create image for `debian` and `alpine`?

I managed to get tag available for version 2.11 of omniDB: https://github.com/OmniDB/OmniDB/issues/695
I tried to create a dockerFile on `debian` for it but Web GUI works into the container, and I get a connection reset after exposure... I will try to correct it quickly, and test on `alpine`

**Comment by taivokasper** [comment]
Hi Thomas,

I decided to stop publishing the debian version. User of this container probably doesn't care on what OS it runs on as long as it doesn't waste a lot of disk space.

I did do tiny changes to your PR and released it under tag [v2.10.0_2-alpine_3.8](https://hub.docker.com/r/taivokasper/omnidb/tags/).

I also published tag OmniDB v2.11.0-alpine_3.8. It seems to work just fine. Thank you for getting OmniDB 2.11.0 tagged!

**Pull request merged**